content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,361 @@
-three lots available.
+In this example, there are three lots of white shirts. The shirts are from the All/Clothes category,
+where :guilabel:`FIFO` is set as the removal strategy. In the :guilabel:`Inventory Valuation
+Report`, the three different receipts are listed with the amounts.
 
 .. image:: removal/inventory-valuation.png
    :align: center
-   :alt: view of the white shirt lots inventory valuation.
+   :alt: View of the lots of white shirts in the inventory valuation report.
 
-The lot 000001 contains 5 shirts, 000002 contains 3 shirts, and 000003 contains 2. As it can be
-seen above, 000001 has entered the stock first. Let’s create a sale order of six white shirts
-to check that products from that lot are the first ones to move out.
+Lot 000001 contains five shirts, lot 000002 contains three shirts, and lot 000003 contains two
+shirts.
 
-On the delivery order linked to the picking, you can see that the oldest lot numbers have been
-reserved thanks to the *FIFO* strategy.
+To see the removal strategy in action, go to the :menuselection:`Sales app` and click
+:guilabel:`Create` to create a sales order. Next, select a :guilabel:`Customer` from the drop-down
+menu. Then click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab. Select a product (for
+this example, the :guilabel:`White Shirt`) from the drop-down menu, or type in the name of the
+product in the field. Enter a quantity (for this example, `6.00`) in the :guilabel:`Quantity` field,
+then click :guilabel:`Save`, then click :guilabel:`Confirm`.
+
+Once the sales order is confirmed, the delivery order will be created and linked to the picking, and
+the oldest lot numbers will be reserved thanks to the :abbr:`FIFO (First In, First Out)` strategy.
+All five shirts from lot 000001 and one shirt from lot 000002 will be selected to be sent to the
+customer.
 
 .. image:: removal/reserved-lots-FIFO.png
    :align: center
-   :alt: two lots being reserved for sell with the FIFO strategy.
+   :alt: Two lots being reserved for a sales order with the FIFO strategy.
 
 LIFO (Last In, First Out)
 -------------------------
 
-With a *LIFO* strategy, that’s quite the opposite. In fact, the products that are brought the
-last move out the first. It is mostly used in case of products without a shelf life.
+The :abbr:`LIFO (Last In, First Out)` removal strategy works in the **opposite** manner from the
+:abbr:`FIFO (First In, First Out)` strategy. With this method, the products that are received
+**last** are moved out first. This method is mostly used for products without a shelf life, and no
+time-sensitive factors, such as expiration dates.
+
+In this example, there are three lots of cinder blocks. The blocks are from the All/Building
+Materials category, where :guilabel:`FIFO` is set as the removal strategy. In the
+:guilabel:`Inventory Valuation Report`, the three different receipts are listed with the amounts.
```

Comment:
The first half of this example is not part of an active instruction, so using italics rather than guilabel
```suggestion
In this example, there are three lots of cinder blocks. The blocks are from the *All/Building
Materials* category, where *FIFO* is set as the removal strategy. In the :guilabel:`Inventory
Valuation Report`, the three different receipts are listed with the amounts.
```